.github/workflows/codeql.yaml

```diff
@@ -1,28 +1,39 @@
 name: "CodeQL"
 on:
-  workflow_dispatch:
   push:
-    branches: [ master ]
+    branches:
+    - master
+    - release/v1.4
```

Comment:
Do we expect to make any more release/* branches? If so, why not leverage a wildcard here?

Reply:
fixed